### DISCUSSION TITLE
the other Fabric mods (like Sodium) in relation to Iris — what do they do?

### DISCUSSION THREAD

**Discussion by gaeneric** [IrisShaders/Iris#1145] [Q&A] [answered]
Hello, I wish to use more Fabric mods, and for aesthetic purposes, i wish to have Iris as well. I understand that Sodium helps with performance with Iris, but I'm not sure about all the other mods with like indium lithium etc.
Is that anyone who can explain them to me, and what they do, if they're compatible or not, and what is best to put together?

**Comment by DioEgizio** [comment, marked as answer]
Sodium - drastically improves rendering performance
Indium - adds frapi support to sodium
Lithium - optimizes game logic
Starlight/Phosphor - optimizes lighting engine
Hydrogen/ferritecore - ram
Krypton - networking
LazyDFU - faster boot
Smooth Booth - faster boot
Enhanced Block Entities - optimizes block entities


I recommend putting all them together but note that starlight and phosphor are incompatible as they do the same thing, try both and see

**Reply by SHUKLASHRI** [reply]
will it work with iris??
